src/commercial.ts

```diff
@@ -18,13 +18,18 @@ void (async () => {
 		!commercialFeatures.adFree
```

Comment:
Could this condition be moved out into a named function that describes this particular state?

Eg.

```suggestion
		function shouldRunConsentless(consentState): boolean {
				return window.guardian.config.switches.optOutAdvertising &&
				consentState.tcfv2 &&
				!getConsentFor('googletag', consentState) &&
				!commercialFeatures.adFree;
		}
		...
		if (shouldRunConsentless(consentState) {
```

It would improve readability and make it clearer what all these conditions actually add up towards.